source/standards/standards/angular-web-performance.html.md.erb

```diff
@@ -0,0 +1,311 @@
+
+
+#### Improved Core Web Vitals
+
+With SSR, the largest content elements (images, text blocks) are usually included in the initial HTML, leading to faster [Largest Contentful Paint (LCP)](https://web.dev/articles/lcp) times. SSR also minimizes layout shifts that can occur when the Angular application is bootstrapped on the client-side and dynamically loads content. Reducing the [Cumulative Layout Shift (CLS)](https://web.dev/articles/cls) of a page leads to a more visually stable experience.
+
+
+#### Better Search Engine Optimization (SEO)
+
+Search engine crawlers can easily index the fully rendered HTML content generated by SSR, which is [important for SEO visibility](https://web.dev/articles/pass-lighthouse-seo-audit) and search engine crawlability. SSR ensures that social media platforms can accurately display previews of shared pages, as they can access the complete HTML structure and content, leading to improved Social Sharing.
+
+
+#### Enhanced User Experience
+
+As mentioned above, the faster initial rendering and lower CLS contribute to a smoother and more engaging user experience, especially on slower connections or devices. This improves [perceived performance](https://developer.mozilla.org/en-US/docs/Learn/Performance/Perceived_performance) for users. SSR makes content available to users with [JavaScript disabled or with network connectivity issues](https://www.gov.uk/service-manual/technology/using-progressive-enhancement#do-not-assume-users-turn-off-css-or-javascript). Users using screen readers will see an improvement in accessibility and inclusivity.
+
+
+#### Reduced Client-Side Load
+
+SSR shifts some of the rendering work to the server, reducing the amount of JavaScript that needs to be executed on the client-side, potentially leading to faster interactions and reduced battery consumption on mobile devices. Less JavaScript Execution on the client is always a great performance target to aim for.
+
+
+#### Considerations
+
+There are some important considerations that should be thought about before using SSR. SSR increases the server-side workload, as the server needs to render each page for each user. This can impact scalability and requires careful server-side optimisation. Caching the rendered pages should be a high priority to reduce the server-side workload. It’s also important to consider SSR can add significant complexity to Angular applications.
+
+
+#### Summary
+
+Using Server-side Rendering in an Angular application comes with some trade-offs. But it can significantly improve performance, SEO, and user experience. These improvements are especially important for [public-facing websites and applications where performance](https://technology.blog.gov.uk/2019/04/18/why-we-focus-on-frontend-performance/) and SEO are crucial.
+
+
+### Static Site Generation (SSG)
```

Comment:
Do we have any angular applications where this is applicable?

The content looks great but I'm concerned about these if we don't:

1. cognitive overhead of longer / more docs 
2. maintenance of additional docs 
3. risk of rot

Reply:
I'm certainly willing to add these points you raise as questions within this guidance if that helps? e.g. "Before adopting SSR or SSG, evaluate if the added maintenance, documentation, and developer overhead outweigh potential benefits. Consider if the functionality risks becoming redundant."

Reply:
I've now added this feedback as a separate line under the considerations header.